packages/language-server/src/utils/node/isPathInsideTemplateEmbedding.ts

```diff
@@ -0,0 +1,25 @@
+import { templateUsingFunctions, templateUsingStatements } from "constants/template-usage";
+import { SyntaxNode } from "web-tree-sitter";
+import { parseFunctionCall } from "./parseFunctionCall";
+
+export const isPathInsideTemplateEmbedding = (node: SyntaxNode): boolean => {
+	if (node.type !== 'string' || !node.parent) {

```

Comment:
> \t

spaces > tabs

Reply:
just kidding. Will introduce [biome](https://biomejs.dev/) after this gets merged